Userland/Libraries/LibGUI/TabWidget.cpp

```diff
@@ -635,6 +635,17 @@ void TabWidget::context_menu_event(ContextMenuEvent& context_menu_event)
     }
 }
 
+void TabWidget::doubleclick_event(MouseEvent&)
+{
+    for (auto& tab : m_tabs) {
+        auto* widget = tab.widget;
+        deferred_invoke([this, widget] {
+            if (on_double_click && widget)
+                on_double_click(*widget);
+        });
```

Comment:
that `widget` won't change, how about
```suggestion
        if (auto* widget = tab.widget) {
            deferred_invoke([this, widget] {
                 if (on_double_click)
                     on_double_click(*widget);
            });
        }

Reply:
Good idea, thanks for the super fast feedback as well :)